.spelling

```diff
@@ -900,3 +900,11 @@ INPUT_TYPES.number
 inputLength
 onSubmit
 otp
+currentIndex
+totalIndicators
+showNav
+indicatorLabel
+prevNavLabel
+nextNavLabel
+overImage
+default
```

Comment:
This is a correct word, I think it shouldn't be here

Reply:
Have removed this, thank you very much!